prifi-lib/relay/relay.go

```diff
@@ -372,20 +373,23 @@ func (p *PriFiLibRelayInstance) upstreamPhase2b_extractPayload() error {
 
 	//disruption-protection
 	if p.relayState.DisruptionProtectionEnabled {
+		// Getting and checking the b_echo_last flag
```

Comment:
By the way this is the old protocol. Now the relay starts the blame when he gets the NIZKPoK, not b_echo